src/Auth0.AspNetCore.Authentication/TokenClient.cs

```diff
@@ -14,7 +14,7 @@ internal class TokenClient
         private readonly HttpClient _httpClient;
         private readonly JsonSerializerOptions _jsonSerializerOptions = new JsonSerializerOptions()
         {
-            IgnoreNullValues = true,
+            DefaultIgnoreCondition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull
```

Comment:
`IgnoreNullValues` is obsolete, see https://learn.microsoft.com/en-gb/dotnet/fundamentals/syslib-diagnostics/syslib0020

> To ignore null values when serializing, set [DefaultIgnoreCondition](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializeroptions.defaultignorecondition#system-text-json-jsonserializeroptions-defaultignorecondition) to [JsonIgnoreCondition.WhenWritingNull](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonignorecondition#system-text-json-serialization-jsonignorecondition-whenwritingnull). For more information, see https://github.com/dotnet/runtime/issues/39152.